airflow/providers/databricks/hooks/databricks_base.py

```diff
@@ -248,9 +248,11 @@ def _get_aad_token(self, resource: str) -> str:
                         azure_ad_endpoint = self.databricks_conn.extra_dejson.get(
                             "azure_ad_endpoint", AZURE_DEFAULT_AD_ENDPOINT
                         )
+                        azure_databricks_proxy = self.databricks_conn.extra_dejson.get("databricks_proxy")
                         resp = requests.post(
                             AZURE_TOKEN_SERVICE_URL.format(azure_ad_endpoint, tenant_id),
                             data=data,
+                            proxies=azure_databricks_proxy,
```

Comment:
If we're doing that for AAD tokens, shouldn't we do it for "normal" HTTP requests for APIs themselves? Or we may need to specify which `requests` calls should be proxied.

Reply:
You're probably right. Which direction do you think is the most sensible?